Modules and packages/Import module 2/task.md

```diff
@@ -0,0 +1,15 @@
+## Import module 2
+
+You can import not only functions: you can import classes or even other modules. It is customary but not required to place all 
```

Comment:
You can import not only functions but also classes and even other modules. It is customary, though not required, to place all